Loop/Info.plist

```diff
@@ -2,6 +2,15 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>Loop</string>
```

Comment:
I did some searching and couldn't find a canonical repository, and since "loop" is 4 characters I'm positive there's a conflict out there somewhere. I just pushed a change to use MAIN_APP_BUNDLE_IDENTIFIER, so the URL now looks like: "com.loopkit.Loop://".

It works in the simulator. Haven't tested it on my device yet.